src/arr.ts

```diff
@@ -208,56 +217,74 @@ function getRelevantArrInstances(mediaType: MediaType): string[] {
 	switch (mediaType) {
 		case MediaType.SEASON:
 		case MediaType.EPISODE:
-			return sonarr ?? [];
+			return sonarr;
 		case MediaType.MOVIE:
-			return radarr ?? [];
+			return radarr;
+		case MediaType.ANIME:
+		case MediaType.OTHER:
+			return [...sonarr, ...radarr];
 		default:
 			return [];
 	}
 }
 
-export async function scanAllArrsForExternalIds(
+export async function scanAllArrsForMedia(
 	searchee: Searchee,
 	mediaType: MediaType,
-): Promise<Result<ExternalIds, boolean>> {
+): Promise<Result<ParsedMedia, boolean>> {
 	const uArrLs = getRelevantArrInstances(mediaType);
+	if (uArrLs.length === 0) {
+		return resultOfErr(false);
+	}
 	try {
+		const title =
+			mediaType === MediaType.OTHER
+				? cleanseSeparators(
+						stripExtension(searchee.name)
+							.replace(RELEASE_GROUP_REGEX, "")
+							.replace(RESOLUTION_REGEX, "")
+							.replace(REPACK_PROPER_REGEX, ""),
+					)
+				: searchee.name;
+		let error = new Error(`No ids found for ${title} [${mediaType}]`);
 		for (const uArrL of uArrLs) {
-			const externalIds = await getExternalIdsFromArr(searchee, uArrL);
-			if (Object.values(externalIds).some(isTruthy)) {
-				logArrQueryResult(externalIds, searchee.name, mediaType);
-				return resultOf(externalIds);
+			const name =
+				mediaType === MediaType.OTHER &&
+				getRuntimeConfig().sonarr.includes(uArrL)
+					? `${title} S00E00` // Sonarr needs season or episode
```

Comment:
It does not matter if it exists in the metadata or not, as it is only used to parse.

You can test this by finding a series without a special and entering 'Series.S00E00.' into the "Test Parsing" spot on the series index. 

It will need to be in your LIbrary (added) though.